### ISSUE TITLE
Cannot access cheatfile

### ISSUE THREAD

**Issue by edoluck90** [tomvita/Breeze-Beta#1]
when try to Load Cheat from DB the breeze report "Cannot access cheatfile"
titles/010007B010FCC000/cheats/4EEA2970DF38ECE1.txt
the game was sniper elite 4

**Comment by Ikenowski** [comment]
No matter how I load the files, the zip and load from DB or place the titles folder directly into the cheats folder, I receive this same error. I have the latest cheat files downloaded and copied onto my SD card.

**Comment by drewskidrewsgv** [comment]
I’m having this exact same issue.

**Comment by GormAuslander** [comment]
It was working for me for a little while, but suddenly gave me this issue on Skyward Sword. I can still access cheats for BoTW. I tried a fresh install from github and it caused atmosphere to crash when I tried to run it. I had to visit sdsetup.com to get a version of breeze that would launch again. Still not fixed after reinstall.

**Comment by Twiizzers** [comment]
Yes Same Problem Edizon and Breeze cant load any cheats / Edizon: No cheats Loading / Breeze beta18 -->cannot access cheatfile

**Comment by marese310** [comment]
Have anyone found a fix for this yet thanks in advance if you can share

**Comment by noah1gamble** [comment]
same issue

**Comment by nolimet** [comment]
Try and update the internal data base used by the application. You can find the latest version of the cheat in here. 
Or you can open the app. (without starting a game) then go to "Download" then "Check for cheat database update". After these short steps it will go look for an update of the cheat database. If there is one you can now presses "Install cheat database update"

**Comment by Spooky-Beetle** [comment]
OP, the issue appears to be that Sniper Elite 4 is not included in the cheat code database. Otherwise, for any game that is not working, make sure the correct cheat file is put in the cheat folder on your device. For the games I am receiving this error for, it appears the wrong cheat txt file is being referenced/the wrong one was placed in the game's cheat folder. Try putting in the correct cheat txt file from the database.
